enterprise/management/commands/create_enterprise_course_enrollments.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+"""
+Django management command for creating EnterpriseCourseEnrollment records.
+"""
+from __future__ import absolute_import, unicode_literals
+
+import logging
+
+from edx_rest_api_client.exceptions import HttpClientError
+from requests.exceptions import ConnectionError, Timeout
+from slumber.exceptions import SlumberBaseException
+
+from django.core.management import BaseCommand, CommandError
+
+from enterprise.api_client.lms import CourseApiClient, EnrollmentApiClient
+from enterprise.models import EnterpriseCourseEnrollment, EnterpriseCustomer
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Creates EnterpriseCourseEnrollment records (if they do not already exist) using the provided
+    EnterpriseCustomer UUID for users enrolled in the provided list of courses.
+    """
+    help = 'Create EnterpriseCourseEnrollment records for all users enrolled in the provided list of course runs.'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-e',
+            '--enterprise_uuid',
+            action='store',
+            dest='enterprise_uuid',
+            default=None,
+            help='EnterpriseCustomer UUID.'
+        )
+        parser.add_argument(
+            '-c',
+            '--courses',
+            action='store',
+            dest='courses_ids',
+            default=None,
+            help='Comma-delimited string of course IDs.'
+        )
+
+    def handle(self, *args, **options):
+        enterprise_uuid = options.get('enterprise_uuid')
+        course_ids = [i.strip() for i in options.get('courses_ids', '').split(',')]
```

Comment:
Right -- `id` is taken but `ids` should be good -- in fact we are correctly working with the plural here. Or does Pylint also complain about `ids`?